### ISSUE TITLE
Feature Request: Stickers

### ISSUE THREAD

**Issue by trymeouteh** [oxen-io/session-android#431]
Please add Stickers to Session. Many chat apps have stickers such as Telegram, Signal and Element.

**Comment by Esdex** [comment]
Agreed. Stickers are an integral part of communication. Looking forward to adding them to the app.
